.erb-lint.yml

```diff
@@ -102,7 +102,6 @@ linters:
       - '*/app/views/users/verify_password/new.html.erb'
       - '*/app/views/users/verify_personal_key/new.html.erb'
       - '*/app/views/users/webauthn_setup/delete.html.erb'
-      - '*/app/views/users/webauthn_setup/new.html.erb'
```

Comment:
We need to rebase this branch (or merge main), since as of #5683 we're enforcing ERBLint on all files.

Reply:
Rebased and pushed just now.